assets/src/apps/delivery/layouts/deck/LessonFinishedDialog.tsx

```diff
@@ -17,15 +18,20 @@ const LessonFinishedDialog: React.FC<LessonFinishedDialogProps> = ({
   const [isOpen, setIsOpen] = useState(true);
   const isPreviewMode = useSelector(selectPreviewMode);
   const resourceAttemptGuid = useSelector(selectResourceAttemptGuid);
+  const { graded } = useDeliveryElementContext();
```

Comment:
oh hmm are you sure `useDeliveryElementContext` is working? I think it says that it needs to have that provider wrapped, and also the DeliveryElement is an activity (screen) not the whole lesson. I have to check but now that I think about it, not sure if we have that graded value passed in or not